common/src/types/token.rs

```diff
@@ -62,126 +113,155 @@ pub static TICKER_NAMES: &[(
 )] = &[
     // L1
     (
-        "WBTC",
+        WBTC_TICKER,
         ExchangeTicker::Renamed("BTC"),
         ExchangeTicker::Renamed("BTC"),
         ExchangeTicker::Renamed("BTC"),
         ExchangeTicker::Renamed("BTC"),
     ),
     (
-        "WETH",
+        WETH_TICKER,
         ExchangeTicker::Renamed("ETH"),
         ExchangeTicker::Renamed("ETH"),
         ExchangeTicker::Renamed("ETH"),
         ExchangeTicker::Renamed("ETH"),
     ),
     (
-        "BNB",
+        BNB_TICKER,
         ExchangeTicker::Same,
         ExchangeTicker::Unsupported,
         ExchangeTicker::Unsupported,
         ExchangeTicker::Same,
     ),
     (
-        "MATIC",
+        MATIC_TICKER,
         ExchangeTicker::Same,
         ExchangeTicker::Same,
         ExchangeTicker::Same,
         ExchangeTicker::Same,
     ),
     // LSDs
-    ("LDO", ExchangeTicker::Same, ExchangeTicker::Same, ExchangeTicker::Same, ExchangeTicker::Same),
-    // Stables
     (
-        "USDC",
+        LDO_TICKER,
+        ExchangeTicker::Same,
         ExchangeTicker::Same,
-        ExchangeTicker::Renamed("USD"), // USDC has 1:1 peg with USD on Coinbase
```

Comment:
The USD- & USDC-quoted markets are literally the same on Coinbase, they are aliased to one another in the API. So it's simpler to just use the USDC ticker in Coinbase (and we don't need to convert a USDC-quoted pair through a USD quote down the line)